source/includes/_architecture_deep_linking.md

```diff
@@ -123,7 +144,33 @@ This URL will take a user to the automated program creation screen.
 > values e.g. "New York" vs "NewYork"
 
 ```json
-[{"column":"agent_name","rules":[{"operator":"eq","value":"Jim FakeName"},{"operator":"eq","value":"Jane FakeName"}]},{"column":"price","rules":[{"operator":"range","value":{"gt":1000000,"lt":2000000}}]}]
+[
```

Comment:
this annoyingly get's auto formatted every time. we should just remove this example code and change the text right above this to something like "Optionally remove extra white space and line breaks"

Reply:
decided to change the code block from json to text